awx/ui_next/src/screens/Job/WorkflowOutput/WorkflowOutput.jsx

```diff
@@ -0,0 +1,222 @@
+import React, { useState, useEffect } from 'react';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import styled from 'styled-components';
+import { CardBody as PFCardBody } from '@patternfly/react-core';
+import { layoutGraph } from '@util/workflow';
+import ContentError from '@components/ContentError';
+import ContentLoading from '@components/ContentLoading';
+import { WorkflowJobsAPI } from '@api';
+import WorkflowOutputGraph from './WorkflowOutputGraph';
+
+const CardBody = styled(PFCardBody)`
+  height: calc(100vh - 240px);
+  display: flex;
+  flex-direction: column;
+`;
+
+const Toolbar = styled.div`
+  height: 50px;
+  background-color: grey;
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-flow: column;
+  height: 100%;
+`;
+
+const fetchWorkflowNodes = async (jobId, pageNo = 1, nodes = []) => {
+  try {
+    const { data } = await WorkflowJobsAPI.readNodes(jobId, {
+      page_size: 200,
+      page: pageNo,
+    });
+    if (data.next) {
+      return await fetchWorkflowNodes(
+        jobId,
+        pageNo + 1,
+        nodes.concat(data.results)
+      );
+    }
+    return nodes.concat(data.results);
+  } catch (error) {
+    throw error;
```

Comment:
No point in catching if we're just going to throw. That said, we should probably handle this _somehow_. Perhaps a content error?

Reply:
Should be addressed in https://github.com/ansible/awx/pull/5712